scripts/install-kabanero-foundation.sh

```diff
@@ -134,3 +124,16 @@ if [ "$ENABLE_KAPPNAV" == "yes" ]
 then
   oc apply -f https://raw.githubusercontent.com/kabanero-io/kabanero-operator/${KABANERO_BRANCH}/deploy/optional.yaml --selector=kabanero.io/component=kappnav
 fi
+
+# Install complete.  give instructions for how to create an instance.
+GITHUB_RAW_URL=https://raw.githubusercontent.com/kabanero-io/kabanero-operator/${KABANERO_BRANCH}/config/samples
+if curl --output /dev/null --silent --head --fail "${GITHUB_RAW_URL}/default.yaml"; then
+    SAMPLE_KAB_INSTANCE="${GITHUB_RAW_URL}/default.yaml"
+else
+    SAMPLE_KAB_INSTANCE="${GITHUB_RAW_URL}/full.yaml"
+fi
```

Comment:
Why are there two cases here to handle both default and full yaml?

I thought we were just going to default yaml

Reply:
full.yaml includes cliServices-->image: which should only be visible to our dev team, although both files are present in the repo

Reply:
KABANERO_BRANCH="${KABANERO_BRANCH:-0.2.0-rc.1}"

how is this supposed to work when CHAMP is working with a cloned repo?

Reply:
The default/full yaml check is because version 0.1.2 does not have a `default.yaml`.  So if the user changes KABANERO_BRANCH to 0.1.2 then the URL we give them won't work.

If Champ is using a cloned repo then he is going to ignore the message at the end and create his own Kabanero CR instance that uses his cloned repo.  That was the whole point of this PR, I thought.  If something is still not clear for Champ, please let me know

Reply:
OK, is there anyway to echo am additional message to apply your cloned repo URL if you are working with you're own cloned copy?

I know it sounds obvious, but just trying to be as clear as possible

Reply:
e.g.

echo modify `SAMPLE_KAB_INSTANCE` to point to cloned, modified default.yaml, if pointing to a cloned, curated collection
echo "oc apply -f ${SAMPLE_KAB_INSTANCE}"

Reply:
or just a comment explaining

Reply:
How about:
```
echo "The installation script is complete.  You can now create an instance"
echo "of the Kabanero CR.  If you have cloned and curated a collection set,"
echo "apply the Kabanero CR that you created.  Or, to create the default "
echo "instance:"
echo "oc apply -n ${namespace} -f ${SAMPLE_KAB_INSTANCE}"
```